README.md

```diff
@@ -61,7 +61,7 @@ and the intended outline is:
 
 ## How To Contribute
 
-All pages in the tutorials are built using the Just the Docs format. Therefore, **each Markdown file must include the following YAML configuration at the top**:
+All pages in the tutorials are built using the Jekyll and the Jekyll using the Just the Docs template. Therefore, **each Markdown file must include the following YAML configuration at the top**:
```

Comment:
You could say:  `l pages in the tutorials are built using the Jekyll using the Just the Docs template.`